specs/StreetLighting/StreetlightGroup/schema.json

```diff
@@ -0,0 +1,121 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "id": "https://fiware.github.io/dataModels/specs/StreetLighting/Streetlight/schema.json",
+  "title": "FIWARE - Street Lighting / Streetlight Group",
+  "description": "A Street light group",
+  "type": "object",
+  "allOf": [
+    { "$ref": "https://fiware.github.io/dataModels/common-schema.json#/definitions/GSMA-Commons" },
+    { "$ref": "https://fiware.github.io/dataModels/common-schema.json#/definitions/Location-Commons" },
+    { "$ref": "https://fiware.github.io/dataModels/common-schema.json#/definitions/PhysicalObject-Commons" },
+    {
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": [
+            "StreetlightGroup"
+          ]
+        },
+        "areaServed": {
+          "type": "string"
+        },
+        "powerState": {
+            "type": "string",
+            "enum": [
+                "on",
+                "off",
+                "low",
+                "bootingUp"
+            ]
+        },
+        "refStreetlightControlCabinet": {
+          "$ref": "https://fiware.github.io/dataModels/common-schema.json#/definitions/EntityIdentifierType"
+        },
+        "dateLastSwitchingOn": {
+          "type": "string",
+          "format": "date-time"
+        },
+        "dateLastSwitchingOff": {
+          "type": "string",
+          "format": "date-time"
+        },
+        "switchingOnHours": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "from": {
+                "type": "string",
+                "format": "date-optionally-yearless"
```

Comment:
probably here you need to use `oneOf` so that both a date and a `ext-date-yearless` can be used .